symfony/.php-version

```diff
@@ -0,0 +1 @@
+8.0
```

Comment:
Voor Symfony Webserver? Die zit in root van project niet in /symfony => https://github.com/icapps/php-boilerplate/blob/master/.php-version

Reply:
Is het niet logischer deze in de project root te staan hebben?
Handiger omdat je dan op 1 plek al je commando's runt en niet steeds `$ cd symfony` hoeft te doen?

If you have multiple PHP versions installed on your computer, you can tell Symfony which one to use creating a file called .php-version at the project root directory

Parent root staat in de docs:

The Symfony server traverses the directory structure up to the root directory, so you can create a .php-version file in some parent directory to set the same PHP version for a group of projects under that directory.

Reply:
Die zijn gescheiden met een reden. In root zit alles mbt Docker, deployment, CI/CD... En de .php-version om idd correcte PHP versie in te stellen per project, maar is enkel van toepassing op local development. Symfony map is enkel en alleen het symfony project en wordt ook zo gedeployed